apps/web/modules/ui/components/file-input/lib/action.ts

```diff
@@ -0,0 +1,26 @@
+"use server";
+export const convertHeicToJpeg = async (file: File): Promise<File | null> => {
```

Comment:
the server action is not secured and will be a publicly available endpoint to convert heic to jpeg. We should make this private and at least check for an active Formbricks session.